vignettes/biocDockstore.Rmd

```diff
@@ -0,0 +1,119 @@
+---
+title: "Dockstore and Bioconductor for AnVIL"
+author: "BJ Stubbs, S Gopaulakrishnan, Nitesh Turaga, Martin Morgan, Vincent J. Carey, stvjc at channing.harvard.edu"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    highlight: pygments
+    number_sections: yes
+    theme: united
+    toc: yes
+  BiocStyle::pdf_document:
+    toc: yes
+    number_sections: yes
+---
+
+# Introduction: Basic concepts of Dockstore and Bioconductor
+
+```{r setup,echo=FALSE}
+suppressPackageStartupMessages({
+library(AnVIL) # get from https://github.com/Bioconductor/AnVIL
+library(rjson)
+library(httr)
```

Comment:
Maybe httr and tidyverse (which is the paradigm I would like to adopt for end-users) should be added to the Depends: field of the DESCRIPTION file?

Reply:
Depending on tidyverse seems to me unwarranted.  tidyverse cannot be loaded without readxl and haven ... broom, etc.  tidyverse_deps() returns 25 packages.  Certain dplyr/magrittr patterns seem ok to me but I would propose import as needed